.github/workflows/update-kernels.yml

```diff
@@ -14,36 +14,29 @@ concurrency:
 jobs:
   update-kernels:
     runs-on: ubuntu-latest
-    container:
```

Comment:
Biggest change: we always run the daily workflows using `main` action instead of tagged one.
This streamlines our development workflow (ie: we won't need to always tag a new kernel-crawler release every time).
cc @EXONER4TED

Reply:
We can always manage to make a release if somebody needs it, anyway.
The docker `falcosecurity/kernel-crawler` image will still be there ;)